packages/react-icons/src/utils/useIconState.tsx

```diff
@@ -1,5 +1,6 @@
 import { FluentIconsProps } from "./FluentIconsProps.types";
 import { makeStyles, mergeClasses } from "@griffel/react";
+import { useTextDirection } from '@griffel/react/TextDirectionContext'
```

Comment:
We'll need to have this function exported from griffel before we can actually use it.